content/desktop/kubernetes.md

```diff
@@ -80,4 +80,4 @@ To turn off Kubernetes in Docker Desktop:
 1. From the Docker Dashboard, select the **Settings** icon.
 2. Select **Kubernetes** from the left sidebar. 
 3. Next to **Enable Kubernetes**, clear the checkbox
-4. Select **Apply & Restart** to save the settings.This stops and removes Kubernetes containers, and also removes the `/usr/local/bin/kubectl` command.
\ No newline at end of file
+4. Select **Apply & Restart** to save the settings.This stops and removes Kubernetes containers, and also removes the `/usr/local/bin/kubectl` command.
```

Comment:
Trying to see what changed here, or was this a newline added?

Reply:
I don't really know, i didn't touch that line.